go/pkg/db/key_data.go

```
@@ -0,0 +1,98 @@
+			UpdatedAtM: r.UpdatedAtM, DeletedAtM: r.DeletedAtM, RefillDay: r.RefillDay,
+			RefillAmount: r.RefillAmount, LastRefillAt: r.LastRefillAt, Enabled: r.Enabled,
+			RemainingRequests: r.RemainingRequests, RatelimitAsync: r.RatelimitAsync,
+			RatelimitLimit: r.RatelimitLimit, RatelimitDuration: r.RatelimitDuration,
+			Environment: r.Environment,
+		},
+		Api: r.Api, KeyAuth: r.KeyAuth, Workspace: r.Workspace,
+		EncryptedKey: r.EncryptedKey, EncryptionKeyID: r.EncryptionKeyID,
+	}
+
+	if r.IdentityTableID.Valid {
+		kd.Identity = &Identity{
```

golangci-lint (2.2.2)

db.Identity is missing fields Environment, Deleted, CreatedAt, UpdatedAt

(exhaustruct)